src/Framework/Microsoft.Internal.Aspnetcore.DotNetApiDocs.Transport/src/Microsoft.Internal.Aspnetcore.DotNetApiDocs.Transport.csproj

```diff
@@ -0,0 +1,74 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
For my own education, why did you choose to use Sdk instead of NoTargets, like we did in runtime and extensions? https://github.com/dotnet/extensions/pull/4254

Reply:
Because we rely on the reference resolution logic from the SDK to populate `ReferenceAssemblyWithReferencePath` - this file is basically a stripped-down version of `Microsoft.Aspnetcore.App.Ref.csproj`. The logic originates from there.

Reply:
nit: Microsoft.Build.NoTargets allows the use of `ReferenceAssemblyWithReferencePath` by setting `NoTargetsDoNotReferenceOutputAssemblies=false`.